doc/licenses/readable-stream-2.3.7/README

```diff
@@ -0,0 +1,8 @@
+readable-stream (https://github.com/nodejs/readable-stream)
+---------------------------------------------
+
+    Version: 2.3.7
+    From: 'Node.js' (https://github.com/nodejs)
```

Comment:
According to the license, this should be "Node.js contributors, Joyent, Inc., and other Node contributors".